controls/roles/validator-import-api/tests/inventory

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
this file isn't necessary, please remove

it's generated by molecule

Reply:
done!